stripe/src/test/java/com/stripe/android/view/PaymentAuthWebViewClientTest.kt

```diff
@@ -47,8 +31,13 @@ class PaymentAuthWebViewClientTest {
             "pi_123_secret_456",
             returnUrl = "stripe://payment_intent_return"
         )
-        paymentAuthWebViewClient.shouldOverrideUrlLoading(webView, url)
-        assertThat(activity.isFinishing)
+
+        val shouldOverrideUrlLoading =
+            paymentAuthWebViewClient.shouldOverrideUrlLoading(webView, url)
```

Comment:
inline this inside assertThat too? ditto to others

Reply:
I want to keep this separate, because it determines whether `activityFinisher` is invoked